packages/bindings-test/src/multitest.rs

```diff
@@ -176,6 +176,11 @@ pub struct OsmosisModule {}
 pub const BLOCK_TIME: u64 = 5;
 
 impl OsmosisModule {
+    fn build_denom(&self, contract: &Addr, sub_denom: &str) -> String {
+        // TODO: validation assertion on sub_denom
+        format!("cw/{}/{}", contract, sub_denom)
```

Comment:
I think it doesn't matter here, but just curious - how could that validation work like? Do you think you could link some reference?

Reply:
Yeah, you are right. In fact, that prefix changed, will update.

Reply:
Here's the reference:
https://github.com/cosmos/cosmos-sdk/blob/8ebd28c9f32ddedda6c00d9bceebc5bfed0a9ed1/types/coin.go#L876-L881
https://github.com/cosmos/cosmos-sdk/blob/8ebd28c9f32ddedda6c00d9bceebc5bfed0a9ed1/types/coin.go#L845-L847

Reply:
Also, the actual validation used is different:
https://github.com/cosmos/cosmos-sdk/blob/2646b474c7beb0c93d4fafd395ef345f41afc251/types/coin.go#L677 (cosmos-sdk v0.45.1)

So, these things change. Will add a reference to the TODO, but perhaps leaving this undone is best.